cli/azd/internal/scaffold/funcs.go

```diff
@@ -177,6 +177,14 @@ func ToDotNotation(s string) string {
 	return fmt.Sprintf("'%s'", s)
 }
 
+// EnvFormat takes an input parameter like `fooParam` which is expected to be in camel case and returns it in
+// upper snake case with env var template, like `${AZURE_FOO_PARAM}`.
+func EnvFormat(src string) string {
+	re := regexp.MustCompile(`([a-z0-9])([A-Z])`)
```

Comment:
Prefer at-most-once static initialization here for compiling regex expressions.

Reply:
fixed it. Thank you